.editorconfig

```diff
@@ -210,6 +210,7 @@ dotnet_code_quality.ca1711.allowed_suffixes = Ex
 dotnet_diagnostic.CS0105.severity = error               # CS0105: Using directive is unnecessary.
 dotnet_diagnostic.CS1573.severity = error               # CS1573: Missing XML comment for parameter
 dotnet_diagnostic.CS1591.severity = error               # CS1591: Missing XML comment for publicly visible type or member
+dotnet_diagnostic.CS1712.severity = error				# CS1712: Type parameter has no matching typeparam tag in the XML comment (but other type parameters do)
```

Comment:
Is this indent misaligned, or is it just the GH viewer?

Reply:
Corrected